modules/exploits/windows/scada/rockwell_factorytalk_rce.rb

```diff
@@ -55,7 +55,12 @@ def initialize(info = {})
           [ 'Rockwell Automation FactoryTalk SE', {} ]
         ],
         'DisclosureDate' => '2020-06-22',
-        'DefaultTarget' => 0
+        'DefaultTarget' => 0,
+        'Notes' => {
+          'Stability' => [ CRASH_SAFE ],
+          'SideEffects' => [ IOC_IN_LOGS ],
+          'Reliability' => [ REPEATABLE_SESSION ]
```

Comment:
Is this a good option given it this involves a race condition and two information leaks? Seems to me there is a small chance that the race condition might be lost here.

Reply:
The exploit runs in a loop; so it should be repeatable https://github.com/rapid7/metasploit-framework/blob/4cab6cd0dba37f011bfdc7153ddfc1a3b846d236/modules/exploits/windows/scada/rockwell_factorytalk_rce.rb#L205-L212